testing/x/json_match.go

```diff
@@ -37,7 +38,19 @@ func AssertJSONMatchesPattern(
 	expectedPattern json.RawMessage,
 	actual json.RawMessage,
 ) {
+	AssertJSONMatchesPatternWithOpts(t, expectedPattern, actual, JSONMatchOptions{})
+}
+
+type JSONMatchOptions struct {
```

Comment:
Might need to move here https://github.com/pulumi/providertest/tree/main/replay

Reply:
Looks like the two packages have already drifted a bit: https://github.com/pulumi/pulumi-terraform-bridge/pull/1496

Should we move the tfbridge tests to use providertest instead and add some form of deprecation message to the module in tfbridge?

Reply:
>Should we move the tfbridge tests to use the replay from providertest instead and add some form of deprecation message to the module in tfbridge? 

Yes that'd be ideal. Separate PR if possible.

Reply:
I've raised https://github.com/pulumi/pulumi-terraform-bridge/issues/1604 and assigned it to myself.